config/s3/s3api/s3_copy_object_range_read.yaml

```diff
@@ -0,0 +1,131 @@
+ test_1:
+  TEST_ID: TEST-36594
+  object_size:
+    start: 0Kb
+    end: 100Kb
+  result_duration: 2h
+  sessions_per_node: 1
+  range_read: 100bytes
```

Comment:
Range read should have specific range. @VikasKumarC, @ketanarlulkar can add your view here.

Reply:
IMO this change / review comment should have come during test case / test plan review. We need to discuss and decide and based on that we can take up another task.
@VikasKumarC  @ketanarlulkar @DhobaleVishal